### DISCUSSION TITLE
Consider adding a new section: Frameworks

### DISCUSSION THREAD

**Discussion by felladrin** [felladrin/awesome-ai-web-search#15] [Ideas]
There was a [PR](https://github.com/felladrin/awesome-ai-web-search/pull/12) requesting the addition of a framework, which brought up the idea of adding a Frameworks section below the Applications.

I still don't know if it will be useful enough to compensate for the work of maintaining it. Alternatively, I could rename the repository to `awesome-ai-web-search-applications` and restrict it to Applications only.

While it's decided, we can build a list of the tools that would fit this category. If you know of any framework related to AI Web Search, please leave it in a comment below.

**Comment by felladrin** [comment]
[JigsawStack AI](https://jigsawstack.com/ai-search)

**Comment by felladrin** [comment]
[Crawl4AI](https://github.com/unclecode/crawl4ai)

**Comment by felladrin** [comment]
[ScrapeGraphAI](https://github.com/ScrapeGraphAI/Scrapegraph-ai)

**Comment by boris-agent** [comment]
[Linkup](https://github.com/LinkupPlatform)

**Reply by felladrin** [reply]
Linking here the issue that contains more info:
- https://github.com/felladrin/awesome-ai-web-search/issues/18

**Comment by felladrin** [comment]
[Tako](https://trytako.com/)

**Comment by alhambra81** [comment]
[AI Hay](https://ai-hay.vn) Perplexity of Vietnam [Top 80 Amazon AWS Generative AI Startups](https://www.washingtonpost.com/creativegroup/stage/aws/start-ups-shaping-the-future-of-generative-ai/)

**Reply by felladrin** [reply]
Ah! Nice! This one is not a framework, but an actual site with LLM integrated with web-search. So it can be added right away.

I've [added it](https://github.com/felladrin/awesome-ai-web-search/commit/eecd473b1518961eff8f7be9a4afa365359ac0c3). Thanks!

**Comment by felladrin** [comment]
[Bright Data](https://brightdata.com/)

**Comment by felladrin** [comment]
[AI Search API](https://aisearchapi.io/)

**Comment by felladrin** [comment]
[WFGY 16 Problem Map](https://github.com/felladrin/awesome-ai-web-search/issues/49)

**Comment by felladrin** [comment]
[Desearch](https://desearch.ai/)

**Comment by felladrin** [comment]
[Olostep](https://www.olostep.com/)

**Comment by felladrin** [comment]
A new section has been added to the README.md ([Tools for AI Agents](https://github.com/felladrin/awesome-ai-web-search/commit/85b81c4cb107675a191ced816295bb83da5965c0)), containing the software/frameworks that fit general-purpose web search.
